app/concerns/wpcc/salary_threshold.rb

```diff
@@ -0,0 +1,9 @@
+module Wpcc
+  module SalaryThreshold
+    extend ActiveSupport::Concern
+
+    def load_config(file:)
+      YAML.load_file(Wpcc::Engine.root.join('config', "#{file}.yml"))
+    end
+  end
+end
```

Comment:
hmm, I'm not sure it should be a module or as part of such a specific namespace as its sole purpose is to load a yml file. Altho used by SalaryThreshold it has nothing to do with SalaryThreshold. I was thinking it could be a class for use by WPCC e.g. FileLoader